security.md

```diff
@@ -0,0 +1,135 @@
+# Security
+
+This is our public security policy, which is a subset of our security practices.
+
+This document exists both for education of our employees as well as for
+reference by our customers.  Changes are welcome and should be reviewed by Eric.
+
+Shorebird is an software application.  Most of our code is open source and
+publicly reviewable on GitHub.  We use Google Cloud for the bulk of our
+infrastructure.
+
+Shorebird only offers a hosted service at this time.  We do not offer [on-prem
+or cloud-prem at this
+time](https://github.com/shorebirdtech/shorebird/issues/485).
+
+## Terms
+
+Throughout this document we will refer to the following terms:
+- Customer / you: A user of Shorebird.
```

Comment:
Should we refrain from `you` and keep to `customer`? No strong argument other than to use a single term per "thing"

Reply:
I'm going to keep it for now, thanks!